src/framework/security/authentication/http.clj

```diff
@@ -0,0 +1,35 @@
+(ns framework.security.authentication.http
+  (:require [buddy.auth :as buddy-auth]
+            [buddy.auth.backends :as buddy-auth-backends]
+            [buddy.auth.backends.httpbasic :as buddy-auth-backends-httpbasic]
+            [buddy.auth.middleware :as buddy-auth-middleware]
+            [buddy.hashers :as buddy-hashers]
+            [framework.security.authentication.jwt :refer [create-token]]))
+
```

Comment:
Why multiple NLs? I know it's a small thing, but we should be consitent throughout the code, one NL is widely used standard and I don't see a reson to break it.

Reply:
What it means NL? Sorry, i did not got your point here

Reply:
new line